domain/src/commonMain/kotlin/com/islamversity/domain/AyaOption.kt

```diff
@@ -0,0 +1,11 @@
+package com.islamversity.domain
+
+inline class AyaOption(val enable : Boolean){
+    companion object {
+        val DEFAULT = AyaOption(false)
+    }
+
+    fun String.toBoolean(){
+        equals("true",true)
+    }
```

Comment:
this is very misleading.
because `"1"` does not equal to `false`.
(booleans are "bad")

if you want to stick with booleans I would suggest something like:
```Kotlin

when(this){
"true" -> true
"false" -> false
null -> false
else -> error("$this is not a real option)"
}
```

Reply:
Yes, you're right, but actually String class has a `toBoolean` method and I used that, this is useless.

Reply:
`string.toBoolean()` is very close to what you had before.
to me it's not really meaningful to have `false` as the fallback for invalid inputs